public/build/main.js

```diff
@@ -300,6 +300,33 @@
 			return uniqueStories;
 		}
 	
+		function sizeHeadlineTextAccordingly() {
+	
+			var headlineEls = Array.prototype.slice.call(document.getElementsByClassName('main-stories__story'));
```

Comment:
Now that we are using babel you could make this a bit more readable by using `Array.from(document.getElementsByClassName('main-stories__story'))`